signatures.md

```diff
@@ -10,40 +10,39 @@ We use signatures in filecoin to verify *something* was done by *someone*. For e
 - Tickets (Signature of proof - [Mining](mining.md)
 - Block signature (Signature over all data in the block - done by block leader)
 
-# What signatures affect
+## What signatures affect
 
 What uses them
 
-- Messages [TODO: Link to messages spec]
+- [Messages](https://github.com/filecoin-project/specs/blob/master/data-structures.md)
 - Block validation
-- Tickets which inform leader election in Expected Consensus (EC) [TODO: link to EC spec]
+- Tickets [TODO: link to EC spec; inform leader election in Expected Consensus (EC)]
 
-Note that messages between actors are not signed this is because messages between actors are always spawned by a message from a user -- which are singed by that user. 
+Note that messages between actors are not signed. This is an unfortunate namespace collision. Messages between actors are always spawned by a message from a user. In this way, messages between actors are containers for signed messages.
```

Comment:
I think this is reversed: "In this way, messages between actors are containers for signed messages.". Signed messages contain a 'cross actor' message. The signature is the way of getting it into the system in the first place, from that point its just a message from the account actor to another actor

Reply:
What do you suggest for better wording?

Reply:
“In this way, signed messages are containers for messages between actors”